src/pages/EntityDetails/Model/ActionLogs/ActionLogs.tsx

```diff
@@ -122,6 +188,11 @@ export default function ActionLogs() {
   }, [modelUUID]);
 
   const tableData = useMemo(() => {
+    const handleOutputSelect = (actionTag: string, type: string) => {
```

Comment:
`type` is a reserved word in typescript strict mode so it might be good to rename this to be ready for the future.

Reply:
To be honest I was surprised that it worked!